src/Components/SingleArticle.jsx

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+export default function SingleArticle() {
+  const URL = useLocation().pathname.slice(10);
+  const [sinArt, setSinArt] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    axios
+      .get(`https://lb-nc-news.onrender.com/api/articles/${URL}`)
+      .then((art) => {
+        setSinArt(art.data.article[0]);
+        setLoading(false);
+      })
+      .catch((err) => {
+        console.log(err, "UNSUCCESSFUL");
+      });
+  }, []);
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <section>
+      <h2>Article ID:</h2> {sinArt.article_id}
```

Comment:
It feels like there's some potential here for using your javascript skills to get this info to the page in a way that's not so long-winded (this way works fine though).